src/components/layout/MenuBar.tsx

```diff
@@ -32,15 +39,34 @@ function MenuBar() {
       </a>
       <a className="m-auto  cursor-pointer" href="https://www.google.com/search?q=papers">
 
-      <img className="m-auto w-1/4 cursor-pointer" src={papers}>
-      </img>
+        <img className="m-auto w-1/4 cursor-pointer" src={papers}>
+        </img>
       </a>
 
-      <a className="m-auto  cursor-pointer" href="https://www.google.com/search?q=plus">
+      <div className='dropDownPlus'>
+      <button className="m-auto cursor-pointer" onClick={toggleDropDown}>
+        <img
+          className="m-auto w-1/4 cursor-pointer"
+          src={isDropDownOpen ? plusOpen : plus}
+          alt="Teal Plus Icon"
+        />
+      </button>
+        
+        {isDropDownOpen && (
+          <ul className="bg-cyan-600 m-4 shadow-md rounded-md">
+            <li className="p-2 hover:bg-gray-200 rounded text-center mb-2">
+              <Link to="/newContact">Create New Contact</Link>
+            </li>
+            <li className="p-2 hover:bg-gray-200 rounded text-center mb-2">
+              <Link to="/companies/new">Create New Company</Link>
+            </li>
+            <li className="p-2 hover:bg-gray-200 rounded text-center">
+              <Link to="/jobapplications/new">Create New Job Application</Link>
+            </li>
+          </ul>
+        )}
+      </div>
```

Comment:
Just a suggestion not a pressing fix : When the dropdown is opened or closed, adding a small animation or transition to enhance the user experience.

Reply:
I completely agree, I attempted to find something on tailwind, but was unable to get a transition to work.
This is the tailwind code I attempted, but was unable to get a result.
``className={`bg-cyan-600 m-4 shadow-md rounded-md transition-all duration-300 ease-in-out transform ${
      isDropDownOpen ? 'scale-100 opacity-100' : 'scale-95 opacity-0'``

Reply:
Does tailwind offer an invisible and visible style where you can always render the elements but conditionally rendering the styles?